cohere/__init__.py

```diff
@@ -8,6 +8,7 @@
 EMBED_URL = 'embed'
 CLASSIFY_URL = 'classify'
 EXTRACT_URL = 'extract'
+WHISPER_URL = 'whisper'
```

Comment:
In blobheart the endpoint is currently 'speech'
We should align to one of these

Reply:
Renamed to speech.